packages/utils/package.json

```diff
@@ -17,7 +17,7 @@
     "docs:watch": "yarn typedoc --watch"
   },
   "engines": {
-    "node": "14 || 16"
+    "node": ">=16"
```

Comment:
Why `>=` ?

Reply:
so we don't break on update again...

Reply:
we make an exception with `utils` as its generally used by projects outside the monorepo, like `hopr-connect`
not being strict with the node engine allows us to avoid the issue we had migrating to `v16` too